operators/advanced_network/advanced_network/manager.cpp

```diff
@@ -192,6 +192,22 @@ Status Manager::allocate_memory_regions() {
   return Status::SUCCESS;
 }
 
+/**
+ * @brief Generic implementation of address_to_port that looks up port in config
+ * This is a final method that cannot be overridden by subclasses.
+ *
+ * @param addr Address string to look up
+ * @return int Port ID or -1 if not found
+ */
+int Manager::address_to_port(const std::string& addr) {
```

Comment:
I don't know if we want a more generic name for this or have more functions. This could be called `interface_to_port` where we take in mlx5, linux names, pcie, or config name. As it is right now this only means PCIe address to me.

Reply:
right, that's a fair idea, but I think out of scope: here we keep the current method (which takes the pcie address). We just implement it once for all managers instead of specific to each of them